AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1235,6 +1235,7 @@ imslp.org#%#AG_onLoad(function(){var b=document.getElementById("sm_dl_wait");if(
 ||cdn.web.tv/asset/javascripts/smothScroll.min.js
 ||serialuk.net/js/copyright.js
 !--- CSS ---
+cafe.daum.net#$#body, body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
```

Comment:
We don't use such rules `example.org##body, html`.
Btw, this should be enough `cafe.daum.net#$#body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }`

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/85945/commits/a9b92910d8d0fc256cb40c632b86d0b4ad370828